doc/source/man.rst

```diff
@@ -32,206 +32,217 @@ rho performs four major tasks:
+  Contains the name of the authentication profile to use to authenticate to the systems that are being scanned. If the individual systems that are being scanned each require different authentication credentials, you can use more than one authentication profile. To add multiple authentication profiles to the network profile, separate each value with a space, for example:
 
   ``--auth first_auth second_auth``
 
-  IMPORTANT: This auth profile must exist before attempting to add the authentication profile to the network profile.
+  **IMPORTANT:** An authentication profile must exist before you attempt to use it in a network profile.
 
 ``--sshport=ssh_port``
 
-  This value is used to support discovery on a non-standard port. Discovery takes place by default on port 22.
+  Sets a port to be used for the scan. This value supports discovery on a non-standard port. By default, the scan runs on port 22.
 
 ``--vault=vault_file``
 
-  This contains the path and filename of the file containing the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path and file name of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
+
+The information in a network profile might change as the structure of the network changes. Use the ``rho profile edit`` command to edit a network profile to accommodate those changes.
+
+Although ``rho profile`` options can accept more than one value, the ``rho profile edit`` command is not additive. If you edit a network profile and pass a new value for an option, the command overwrites all current values for that option. To add additional values to an option, but keep the current values, you must include the option with both its current and new values in the ``rho profile edit`` command.
 
 **rho profile edit --name** *name* **[--hosts** *ip_address* **] [--auth** *auth_profile* **] [--sshport=** *ssh_port* **] [--vault=** *vault_file* **]**
 
-Although all three ``rho profile`` parameters accept more than one setting, the ``rho profile edit`` command is not additive. If a new argument is passed, it overwrites whatever was originally in the profile, it doesn't add a new attribute, even if the parameter is multi-valued. To add or keep multiple values with the edit command, list all parameters in the edit. For example, if a profile was created with an auth value of ``"server1creds"`` and the same profile will be used to scan with both server1creds and server2creds, edit as follows:
+For example, if a network profile contains a value of ``server1creds`` for the ``--auth`` option, and you want to change that network profile to use both the server1creds and server2creds authentication profiles, you would edit the network profile as follows:
```

Comment:
I think `server1creds` and `server2creds` could be wrapped into `` as the first `server1creds` is.

Reply:
There's some nuance here about whether the authentication profile name is being referred to as a specific value in a code string that the user enters or as a concept. I might construct this differently in "real" documentation, but if it looks weird to you in the man page, I'll change it to include the syntax for highlighting.

Change made:  to use both the ``server1creds`` and ``server2creds`` authentication profiles

Reply:
This appears to be a duplicate of a previous comment that you made.

No change made: Duplicate

Reply:
Weird, I am seeing two of these which is why I added the Duplicate info. Anyway, for my sanity:
CHANGE MADE